### PR TITLE
Add staking props

### LINKED ISSUE #3054: Add staking properties
### Problem

HIP-406 specifies multiple new properties to control staking.

### Solution

- [ ] Add global/static property `staking.period=86400` (default unit for all durations is `seconds`).
- [ ] Add global/dynamic property `staking.rewardRate=?`
- [ ] Add global/dynamic property `staking.startThreshold=?`
- [ ] Add global/dynamic property `staking.councilMinStake=?`
- [ ] Add global/dynamic property `staking.councilMaxStake=?`
- [ ] Add global/dynamic property `staking.communityMinStake=?`
- [ ] Add global/dynamic property `staking.communityMaxStake=?`
- [ ] Add global/dynamic property `staking.fees.nodeRewardFraction=?`
- [ ] Add global/dynamic property `staking.fees.stakingRewardFraction=?`

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by anighanta**
**Description**:

**Related issue(s)**:

Fixes #3054 

**Notes for reviewer**:

Just added default testing values to the staking props for now. When Council comes back with the right values we will update all the config files with the correct ones.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java [hunk 1]
```diff
@@ -257,6 +259,10 @@ public Set<String> allPropertyNames() {
 			"scheduling.whitelist",
 			"scheduling.triggerTxn.windBackNanos",
 			"sigs.expandFromLastSignedState",
+			"staking.rewardRate",
```
Comment (Neeharika-Sompalli) [inline hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java:262]: need to add `staking.activeThreshold` property

File: hedera-node/src/main/resources/bootstrap.properties [hunk 2]
```diff
@@ -40,6 +40,8 @@ hedera.shard=0
 ledger.id=0x03
 ledger.numSystemAccounts=100
 ledger.totalTinyBarFloat=5000000000000000000
+staking.periodMins=1440
+staking.rewardHistory.numStoredPeriods=3
```
Comment (tinker-michaelj) [inline hedera-node/src/main/resources/bootstrap.properties:44]: Why not 365 as the default?

File: hedera-node/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java [hunk 3]
```diff
@@ -537,4 +547,24 @@ public int getMaxReturnedNftsPerTouch() {
 	public Gas exchangeRateGasReq() {
 		return exchangeRateGasReq;
 	}
+
+	public double getStakingRewardRate() {
```
Comment (tinker-michaelj) [inline hedera-node/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java:551]: Why is this a `double`?  From the HIP,

> `StakingRewardRate` - the total tinybars earned as staking reward during each staking period (example: 100_000_000L * 1_000_000_000 / 365 to emit rewards at a total rate of one billion hbars per year)

...seems like a `long` would be more appropriate?

Comment (anighanta) [reply hedera-node/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java:551]: ahh, I modified this in the working branch but not here.. Will update this.

**Review by tinker-michaelj — APPROVED** [review]
LGTM

**Pull request merged**